@kiva/kv-components/src/vue/stories/KvUserAvatar.stories.js

```diff
@@ -44,4 +44,10 @@ export const IsSmall = story({
 	isSmall: true,
 });
 
+export const Multiple = story({
+	lenderImageUrl: 'https://www.development.kiva.org/img/s100/26e15431f51b540f31cd9f011cc54f31.jpg',
```

Comment:
Ideally, to simulate what happens in TY page, we need a different image here.

Reply:
Yeah three different images would be ideal.

Reply:
Also ensure adding a different key each repetition, like here: https://github.com/kiva/ui/blob/72afc1451ae021d3c94d07cbbfce0b191601ba90/src/components/Thanks/BorrowerAvatarsContainer.vue#L5